### ISSUE TITLE
unhashable type error workaround anyone ?

### ISSUE THREAD

**Issue by PrithviSharanS** [aigamedev/scikit-neuralnetwork#229]
C:\Python27\Lib\site-packages\sklearn\neural_network\multilayer_perceptron.py:90
4: DataConversionWarning: A column-vector y was passed when a 1d array was expec
ted. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)
WARNING:sknn:WARNING: Expecting `Softmax` type for the last layer in classifier.

unhashable type
